README.md

```diff
@@ -50,7 +50,7 @@ your rcfile should do the trick in most cases.
 
 ```
     The basics
-        goto <magicword>                        Go to shortcut
+        goto [<magicword>...]                   Go to one or more shortcuts
```

Comment:
As this is the most central feature of Goto, I think we ought to devote two lines in the help text for it.
One for a single magicword, and one for multiple.

That would  make both examples more clear I believe

Reply:
Good idea